templates/asciidoctor-compatibility.css

```diff
@@ -53,3 +53,70 @@ b.conum *{color:inherit!important}
 
 /* Override Asciidoctor CSS that causes issues with reveal.js features */
 .reveal .hljs table{border: 0}
+
+/* Columns layout */
+.columns .slide-content {
```

Comment:
Not sure if we should (manually) minify the content here? (ie. remove the extra spaces, the trailing semi-colon and remove line breaks)

Reply:
I'm not fond of aggressive minification. This is for slide decks with heavy images and videos. Shaving a few bytes here isn't going to make much of a difference.

OTOH, DevTools now have prettyfication tools built-in and it's easy to see what CSS is applied where.

I don't have a strong opinion on either. However, we did minify some of the existing CSS already. Your call.

Reply:
I don't think we should do it.